rpmlb/cli.py

```diff
@@ -75,14 +75,14 @@
     'recipe_file',
     type=click.Path(exists=True, dir_okay=False, resolve_path=True),
 )
-@click.argument('recipe_name')
-def run(recipe_file, recipe_name, **option_dict):
-    """Download and build RPMs listed in RECIPE_FILE under RECIPE_NAME
-    (such as 'python33').
+@click.argument('collection_id')
+def run(recipe_file, collection_id, **option_dict):
+    """Download and build RPMs listed on COLLECTION_ID (such as 'python33')
```

Comment:
I believe "under" is the right word to use here (... listed *under* COLLECTION_ID ...).